packages/vue/src/dotlottie.ts

```diff
@@ -128,22 +173,37 @@ export const DotLottieVue = defineComponent({
     onMounted(() => {
       if (!canvas.value) return;
 
+      let shouldAutoplay = props.autoplay;
+
+      if (typeof playOnHover?.value !== 'undefined' && playOnHover.value) {
+        shouldAutoplay = false;
+      }
+
       dotLottie = new DotLottie({
         canvas: canvas.value,
         ...props,
+        autoplay: shouldAutoplay,
       });
 
       intersectionObserver = getIntersectionObserver();
-      resizeObserver = getResizeObserver();
       intersectionObserver.observe(canvas.value);
-      resizeObserver.observe(canvas.value);
+      if (typeof autoResizeCanvas?.value === 'boolean' && autoResizeCanvas.value) {
+        resizeObserver = getResizeObserver();
+        resizeObserver.observe(canvas.value);
+      }
       document.addEventListener('visibilitychange', onVisibilityChange);
```

Comment:
we don't need to freeze on visibility change, as the browser automatically stops the animation loop when the tab is not active. Maybe review the React implementation to match Vue

Reference: https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API#policies_in_place_to_aid_background_page_performance

Reply:
removed